### DISCUSSION TITLE
How to hide domain on user registration?

### DISCUSSION THREAD

**Discussion by yasukawa426** [zitadel/zitadel#3967] [Q&A] [answered]
Is there any way to hide the domain on the Username field during user registration?

![image](https://user-images.githubusercontent.com/62857460/178987892-8ecca089-5cd1-4c21-b062-a37d410e5d6f.png)

Couldn't find any options besides disabling "User Loginname must contain orgdomain".

**Comment by hifabienne** [comment, marked as answer]
We have actually in the settings of the branding in the advanced behavior section a flag "Hide Loginname suffix".
I just tried it by myself and this does only trigger on the login and not on the register.
I have created an issue to track this: https://github.com/zitadel/zitadel/issues/3968

**Reply by yasukawa426** [reply]
Thanks for the quick reply!
